static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -177,6 +177,14 @@ const bootStandard = (): void => {
     // Set adtest query if url param declares it
     setAdTestCookie();
 
+    // Remove the old ad-free cookie - we can delete this line in a few days
+    removeCookie('GU_AFU');
```

Comment:
We're not responding to that cookie anywhere else now. I merged a Fastly change a short while ago to ignore it. This is more about tidying up really, as the new cookie will take precedence. I was wondering if this is the best place for this kind of thing to be done - early enough in the process, and presumably widely executed?